### ISSUE TITLE
Turkish Keyboard @ character not working

### ISSUE THREAD

**Issue by adembulut** [Eugeny/tabby#9757]
Version: 1.0.208
Platform: macOS arm64 23.0.0
Plugins: none
Frontend: xterm-webgl

I am using Turkish keyboard (qwerty). Normally, on my macOS computer, the Options+Q key combination is used for the "@" sign. I have no problem in terminal.app but in tabby.sh this character is not printed

**Comment by holgerstolzenberg** [comment]
Can confirm this happens to me on a german keyboard, hitting "Option + L" this not print a "@" char to terminal.
Copy/pasting "@" works.

Version: 1.0.208
Platform: macOS arm64 14.5 Sonoma (23F79)
Plugins: default installation
Frontend: xterm-webgl

**Comment by holgerstolzenberg** [comment]
Can confirm this happens to all special chars can be reached by "Option" modifier.
This essentially makes it impossible to write certain scripts in `vi` in `tabby`.

**Comment by holgerstolzenberg** [comment]
Most probably a duplicate of #9746

**Comment by holgerstolzenberg** [comment]
Okay - I have to revert my assumption. My problem boils down to the Mac OS X input sources. I added "German - Standard" keyboard input source via global system preferences. This input source behaves differently than "German".
Removing "German - Standard" keyboard input source and adding "German" input source fixed the problem for me.
So for me the issue is not related to `tabby` at all.

I realised the problem when trying the type `@` in other apps, where it also did not work.
